distributed-tracing/node-js-app/loadgen.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+while true; do
+    read -p "Please input API call interval in sec " input
+    if [[ $input =~ ^[0-9]+$ ]] ; then
+        echo "Send requests every $input seconds"
+
+        while [ true ]
+        do
+            quantity=$((1 + RANDOM % 20))
+            curl_cmd="curl -X POST http://localhost:3000/shop/order -H 'Content-Type: application/json' -d '{\"styleName\": \"beachops\", \"quantity\": '$quantity'}'"
+            echo $curl_cmd
+            eval $curl_cmd
+            sleep $input
+            printf "\n"
+        done
+    else
+        echo "Please input an integer "
+    fi
+done
```

Comment:
L10: How is RANDOM defined?

Reply:
To generate a random number in a UNIX or Linux shell, the shell maintains a shell variable named RANDOM. Each time this variable is read, a random number between 0 and 32767 is generated.

https://blog.eduonix.com/shell-scripting/generating-random-numbers-in-linux-shell-scripting/

Reply:
What happens if a developer runs this script without the RANDOM variable defined in the shell script.
I suggest this script informs the developer that this variable is undefined and should define it as a number.

We want to help new developers using our apps as much as possible.

Reply:
RANDOM is a variable provided by the shell, I have not defined it in my shell script.
e.g. ```$(( 1 + RANDOM % 20))``` This is a function that generate the number between 1 to 20